content/documentation/pages/5-feature-guides/1-general/1-applicaiton-metadata.md

```diff
@@ -46,19 +46,19 @@ To use the processor, include a dependency on spring-boot-configuration-processo
 
 ### Whitelisting properties
 
-In addition to this file, Data Flow can make use of an additional file that whitelists application properties that are the most important, usually your own application's `@ConfigurationProperties`.
-Without using a whitelist, the UI and Shell will present the hundreds of [Common Application Properties](https://docs.spring.io/spring-cloud-dataflow/docs/current/reference/htmlsingle/#spring-cloud-dataflow-global-properties) resulting in a poor user experience.
+In addition to this file, Data Flow can make use of an additional file that whiteliststhe most important application properties, usually your own application's `@ConfigurationProperties`.
+Without using a whitelist, the UI and Shell present the hundreds of [Common Application Properties](https://docs.spring.io/spring-cloud-dataflow/docs/current/reference/htmlsingle/#spring-cloud-dataflow-global-properties), resulting in a poor user experience.
```

Comment:
We cannot yet deprecate the file name, but maybe we can describe the term differently?

Reply:
In other places, we have decided to leave the old term until we can change the corresponding file name (or code, in other cases). For consistency, I'll leave it for now, in spite of my itch to change it.